plugins/database/quota/src/org/apache/cloudstack/api/response/QuotaResponseBuilderImpl.java

```diff
@@ -383,31 +383,41 @@ public QuotaTariffVO updateQuotaTariffPlan(QuotaTariffUpdateCmd cmd) {
 
     @Override
```

Comment:
it is removed from QuotaResponseBuilder (why?) but still marked as @Override here. any purpose?